config/cp4waiops-custom/templates/custom-size.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if eq .Values.profile "x-small-custom" }}
```

Comment:
How about using [hasSuffix](https://helm.sh/docs/chart_template_guide/function_list/#hasprefix-and-hassuffix), which can be flexible enough w/ better naming implication, i.e., any profile or profile variant that has suffix `-custom` will be treated as sizing to be customized against a certain baseline. With that, the naming convention of profile value will be:

```
<profile|profile-variant>-custom
```

e.g.: x-small-custom, x-small-idle-custom, etc.

Reply:
Agree to use suffix. Will update accordingly.